guides/source/upgrading_ruby_on_rails.md

```diff
@@ -85,7 +85,7 @@ Rails 6.1. You are encouraged to enable `config.force_ssl` to enforce HTTPS
 connections throughout your application. If you need to exempt certain endpoints
 from redirection, you can use `config.ssl_options` to configure that behavior.
 
-### Purpose in signed or encrypted cookie is now embedded within cookies
+### Purpose metadata in signed or encrypted cookie is now embedded within cookies
```

Comment:
The existing sentence is actually correct, though incomplete, the proposed change makes it sound even more wrong.

This should be:
```
Purpose and expiry metadata is now embedded inside signed and encrypted cookies for increased security.
```

Reply:
Yes, that's actually correct.